packages/design-system-scripts/jest/unit.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = (rootDir) => ({
+  rootDir,
+  testURL: 'http://localhost',
+  setupFiles: [`<rootDir>/${path.relative(rootDir, __dirname)}/polyfills.js`],
+  setupFilesAfterEnv: [`<rootDir>/${path.relative(rootDir, __dirname)}/setupEnzyme.js`],
```

Comment:
I keep getting the following Enzyme error:
```
Enzyme Internal Error: Enzyme expects an adapter to be configured, but found none.
          To configure an adapter, you should call `Enzyme.configure({ adapter: new Adapter() })`
          before using any of Enzyme's top level APIs, where `Adapter` is the adapter
          corresponding to the library currently being tested. For example:

          import Adapter from 'enzyme-adapter-react-15';

          To find out more about this, see http://airbnb.io/enzyme/docs/installation/index.html
```
When I copy the enzyme/adapter config from `setupEnzyme.js` to the top of a test file, the error goes away. Frustratingly, if I add a console.log to `setupEnzyme.js` I can see that the file is being found and run before each test file like its supposed to, but some reason I'm still getting the Enzyme error 😠

Reply:
I couldnt completely solve the issue, the solution im going with for now is to move the `setupEnzyme` file to the root of the directory